Gopkg.toml

```diff
@@ -57,7 +57,7 @@
   name = "github.com/hashicorp/raft-boltdb"
 
 [[constraint]]
-  revision = "bc5b7e5db16683db8ddffa7eef67b53706a4618a"
+  revision = "735c14e2335ad2d8112482794d69629c0517476c"
```

Comment:
I do not think we want to change these here!Isn't it all revisions from 2.4?

Reply:
2.4 uses dependencies.tsv instead of Gopkg.toml
These shas have been ported across from the changes to dependencies.tsv

Reply:
Niice pick up :) Thank you!